ui/cypress/integration/volume/volumelist.spec.js

```diff
@@ -0,0 +1,67 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+});
+
+// Navigation tests
+describe('Volume list', () => {
+  beforeEach(() => {
+    // Visit is automatically prefixed with baseUrl
+    cy.visit('/volumes');
+    cy.stubHistory();
+  });
+
+  it('brings me to the overview tab of the unhealthy volume', () => {
+    // Volume `master-0-alertmanager` has alert.
+    // According to the default sorting rule, it should appear at the first place.
+    cy.wait(['@getPVs', '@getPVCs', '@getAlerts', '@getVolumes']);
```

Comment:
Do we really need to wait for those? If we do, you'll want to rebase and make sure aliases are available as you expect - see [`ffef869` (#2891)](https://github.com/scality/metalk8s/pull/2891/commits/ffef8699ccd12ed6a15837f7cb5b72222ca46978)